src/components/MenuItem.css

```diff
@@ -5,3 +5,14 @@
   margin: 1.25rem;
   text-align: left;
 }
+
+.menu__buttons {
+  border: none;
+  background-color: white;
+  padding: 0 0.5rem 0 0.5rem;
+  font-size: 25px;
+}
+
+.menu__div {
```

Comment:
This name is not really telling much about what is this inside. Normally we discourage using the element name in the className.